docs/how/migrating-graph-service-implementation.md

```diff
@@ -0,0 +1,44 @@
+# Migrate Graph Service Implementation to Elasticsearch
+
+We currently support either Elasticsearch or Neo4j as backend implementations for the graph service. We recommend
+Elasticsearch for those looking for a lighter deployment or do not want to manage a Neo4j database.
+If you started using Neo4j as your graph service backend, here is how you can migrate to Elasticsearch.
+
+## Docker-compose
+
+If you are running your instance through docker locally, you will want to spin up your Datahub instance with
```

Comment:
Can you also add the command to do this using datahub cli?

Reply:
updated